subprojects/problems/src/main/java/org/gradle/api/problems/interfaces/Severity.java

```diff
@@ -13,31 +13,33 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.gradle.internal.reflect.validation;
 
+package org.gradle.api.problems.interfaces;
+
+import org.gradle.api.Incubating;
+
+/**
+ * Severity.
+ *
+ * @since 8.3
+ */
+@Incubating
 public enum Severity {
```

Comment:
💭 Should integration with https://docs.google.com/document/d/19SAbsjnVJr8GsdKC4JqnR96-E6fx6WsuF5Y0I3jJnzE/edit be considered here?